polyply/src/dispatch_gmx.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import vermouth
+import os
+
+class GMXError(Exception):
```

Comment:
You could consider subclassing RuntimeError, or even better ChildProcessError instead, but I'll leave it up to you.
See also https://docs.python.org/3/library/exceptions.html